### ISSUE TITLE
Keyboard shortcut for adding current domain to whitelist

### ISSUE THREAD

**Issue by PureW** [TigerKid001/Owl#26]
Hello,

I've been using Dark Reader for a long time on Chromium which has a keyboard shortcut for adding current domain to the whitelist. This feature is incredibly useful.

Adding keyboard shortcut to Owl would be very useful.

**Comment by TigerKid001** [comment]
Hi, 
If I understand correctly, I think the feature you are asking for is already there. Using Shift+Alt+X, you can disable Owl on the current site. Is this what you were suggesting?

**Comment by PureW** [comment]
Does using Shift+Alt+X persist the disabling so that Owl is disabled on the next visit too?

In my somewhat limited testing that did not seem to be the case.

**Comment by TigerKid001** [comment]
Yes, it adds the site to the whitelist and any subsequent visits to the site use the setting. I'm not sure why it doesn't work as expected in your case. Are you running the latest version v2.2.4? If so, can you please find a reproducable example of this bug?

**Comment by PureW** [comment]
Shift+Alt+X works very well. I believe I must have used  Shift+Alt+D during my first try. 

Perhaps you could add a tooltip about Shift+Alt+X somewhere to explain how to toggle a single site?

**Comment by giosh94mhz** [comment]
Hi,
    please consider this issue as a UI priority. The problem I found -- that led me here -- are:

* no menu by "clicking" on the Owl icon. Also, my expectation was to disable for this website, instead of globally;
* opened plugin preferences to see white/black listing, but there are no add/delete functionality;
* found the shortcut Shift+Alt+D, but it collide with another plugin I have (which I had to disable, since the shortcut is hardcoded).

This was my bad UX. IMO, a menu should unfold from the owl button, stating the status of the plugin (enable/disable), the status of the page (inherit/white/black), then the all the action available with the shortcut hinted. Bonus points, shortcut configurable from about:config.

**Comment by giosh94mhz** [comment]
Hi @CptFoobar, thank you for the great feedback!

I've missed the owl-icon in the bar, I just tough was an on/off indicator.

> The Owl button in the Firefox toolbar serves only as a quick toggle switch.
Maybe this is the source of confusion... If you can sync the two menus it will be great. In my case, I will remove this "redundant" icon, as I love the extension and keep it on by default :)

You can close this issue, if you feel is a duplicate with other open issue.
